source/install/cloud-changelog.md

```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-05-11](#release-2022-05-11)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
-- [Release 2022-03-02](#release-2022-03-02)
+
+## Release 2022-05-11
+
+### Improvements
+
+#### User Interface (UI)
+ - For toggling the channel Info in the right-hand side, a shortcut ``CTRL(or cmd)+Alt+I`` was added.
+ - Added an "Unread Channels" section to the channel switcher and included "Threads" in the results.
+
+#### Performance
+ - Improved the performance of aggregate queries related to Collapsed Reply Threads.
+
+#### Administration
+ - An email is now sent to Cloud Admins after upgrading their workspace.
+ - Added a new ``ConnectionId`` field to ``model.WebsocketBroadcast`` that allows broadcasting a message only to a specific connection.
+ - Timestamps are now enabled in the default audit configuration.
+ - Updated the **System Console** session lengths configurations to use the new hours unit, replacing the day. Also added ``ServiceSettings`` config changes:
+    - ``SessionLengthWebInDays`` changed to ``SessionLengthWebInHours``.
+    - ``SessionLengthMobileInDays`` changed to ``SessionLengthMobileInHours``.
+    - ``SessionLengthSSOInDays`` changed ``SessionLengthSSOInHours``.
```

Comment:
These may not be needed for Cloud changelog - please confirm.

Reply:
The session lengths ones probably not given that the Service Settings are not available in Cloud. Not sure about the other two, I believe audit is not available for Cloud admins either. Is that right @gabrieljackson ?

Reply:
Confirming not relevant to Cloud